content/departments/marketing/content/editorial/editorial-process.md

```diff
@@ -13,41 +13,23 @@ It’s important to have a publishing schedule so that we can deliver relevant c
 
 ## Guidelines for contributing to Sourcegraph’s blog
 
-Please follow these guidelines when contributing to Sourcegraph’s blog.
-
-First, decide how you want to contribute:
-
-### Write the draft yourself
-
-If you have an original article idea that you want to write, start by submitting your pitch via this [Asana form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959) so we can triage appropriately. If you prefer to write your first draft and submit that in place of a pitch, please go ahead! It may still take some time for us to review and schedule your post
-
-**OR**
-
-### We interview you
-
-If you are short on time or less comfortable writing, a member of the content team can interview you and use the transcript to start an outline for you to work from. Please mention if you want to go this route when you fill out your [form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959).
-
-**OR**
+Start by submitting your proposal via a [GitHub issue using the blog post template](https://github.com/sourcegraph/about/issues/new?template=blog_post.md). You can submit a rough draft at the same time if you want to.
 
 ### Just submit an idea
 
-If you have an idea to submit but you don't want to write it, you can still submit the idea via the [Asana form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959), just make a note that you are not proposing to write the post yourself.
+If you have an idea to submit but you don't want to write it, you can still submit the idea via the [GitHub issue template](https://github.com/sourcegraph/about/issues/new?template=blog_post.md), just make a note that you are not proposing to write the post yourself.
 
-### Once you have submitted a pitch
+### Once you have submitted a proposal
 
-1. The Editorial team triages pitches weekly.
-1. An editor will be assigned and will give you feedback on your idea. At this point we may suggest an alternative channel or content format for your idea (guide/case study/social post/etc.)
-1. If the pitch is approved by you and your editor, next we'll decide on a timeline together.
+1. It's up to you if you want to share your proposal issue with the rest of your team or with other peers for feedback.
+1. When you are ready, please @ mention the Content team members listed in the issue template. An editor will be assigned and will give you feedback on your idea.
+1. Next we'll decide on a timeline together and discuss any images or graphics that you want to include so we can organize design work.
 1. First draft is submitted by the first draft deadline.
 1. An editor and/or peer reviewer is completed by the first draft review deadline. Please reference the [peer review template](https://docs.google.com/document/d/1KHFMIGmbmDPsE8EYPAf4DPD6kek_1dRnKf49wG5foU0/edit) when conducting your review. You do not need to answer every reflection question. This is a guide to help you provide useful and actionable feedback to the author.
```

Comment:
@csbailey5t @nickmyyz Wanted a second opinion, I think we should remove this for a few reasons:
- it's very long and may be offputting
- these are the sorts of things we as editors can look for. I would want peer reviewers to focus on fact checking and whether the post makes sense and is compelling technically.
WDYT?

Reply:
@rebeccadee I agree with your reasons. I also agree that peer reviewers should focus on the coherence and impact of the post, too. I'm for removing it!

Reply:
```suggestion
1. An editor and/or peer reviewer is completed by the first draft review deadline. The peer reviewer should focus on the coherence, technical accuracy, and impact of the post.
```